siriuspy/siriuspy/devices/lienergy.py

```diff
@@ -0,0 +1,75 @@
+"""."""
+
+from .device import Device as _Device
+
+
+class LIEnergy(_Device):
+    """Linac Energy Measurement Device."""
+
+    class DEVICES:
+        """Devices names."""
+
+        LI = 'LI-Glob:AP-MeasEnergy'
+        ALL = (LI, )
+
+    _properties = (
+        'Dispersion-SP', 'Dispersion-RB',
+        'IntDipole-Mon', 'Energy-Mon', 'Spread-Mon',
+        'MeasureCtrl-Sel', 'MeasureCtrl-Sts')
```

Comment:
It seems that is missing `'Spectrometer-RB'`and `'Spectrometer-SP'`properties

Reply:
Initially I included these properties but when I tested, there was not connection with them. This is why I remove it.

Reply:
Initially I included and just removed from _properties, I forgot to remove the properties. Thank you!